Sources/TurboNavigator.swift

```diff
@@ -55,8 +55,8 @@ public class TurboNavigator {
 
     // MARK: Internal
 
-    let session: Session
-    let modalSession: Session
+    var session: Session
```

Comment:
Hey @joemasilotti!  what do you think about making these sessions public?  The reason I ask is because if we wanted to use javascript pop up alerts we should have access to the session to set we uiDelegate as far as I can tell from the turbo-ios project
`self.session.webView.uiDelegate = self.session`
I am having trouble wrapping my head around how I would do this with the TurboNavigator instead.

Reply:
You can customize the web view instance in the configuration block. Set the UI delegate there. For example,

```swift
TurboConfig.shared.makeCustomWebView = {
    let configuration = WKWebViewConfiguration()
    // TODO: Customize configuration.
    let webView = WKWebView(frame: .zero, configuration: configuration)
    webView.uiDelegate = self
    return webView
}
```